lib/jekyll-github-metadata/site_github_munger.rb

```diff
@@ -42,10 +42,13 @@ def drop
 
       # Set `site.url` and `site.baseurl` if unset.
       def add_url_and_baseurl_fallbacks!
-        site.config["url"] ||= Value.new("url", proc { |_c, r| r.url_without_path })
+        Jekyll::GitHubMetadata.log :debug, "Adding URL and base URL fallbacks..."
+        site.config["url"] ||= Value.new("url", proc { |_c, r| r.url_without_path })          
+        Jekyll::GitHubMetadata.log :debug, "URL is set to #{site.config["url"]}"
         return unless should_set_baseurl?
 
         site.config["baseurl"] = Value.new("baseurl", proc { |_c, r| r.baseurl })
+        Jekyll::GitHubMetadata.log :debug, "baseurl is set to #{site.config["baseurl"]}"
```

Comment:
Use `base URL` here for consistency with the above.

---

Or use `url` and `baseurl` (lowercase, no spaces) throughout.

Some quotes or backticks would be slightly better then.

```
"`baseurl` is set to #{site.config["baseurl"]}"
```

```
"'baseurl' is set to #{site.config["baseurl"]}"
```